src/components/ItemView.vue

```diff
@@ -34,8 +46,13 @@ const references = computed(() => parseReferences(item.properties?.references ??
         <div class="aux">
             <div class="col">
                 <div class="authors"><ul><li v-for="contact in item.properties?.contacts">{{ contact.name ?? contact.organization }}</li></ul></div>
-                <div class="time" v-if="item?.properties?.start_datetime && item?.properties?.end_datetime">{{ item.properties.start_datetime }} - {{ item.properties.end_datetime }}</div>
-                <div class="time" v-else-if="item?.properties?.datetime">{{ item.properties.datetime }}</div>
+                <div class="time" v-if="start_datetime.iso && end_datetime.iso">
+                    <time :datetime="start_datetime.iso">{{ start_datetime.fmt }}</time> to
+                    <time :datetime="end_datetime.iso">{{ end_datetime.fmt }}</time>
+                </div>
+                <div class="time" v-else-if="datetime.iso">
+                    <time :datetime="datetime.iso">{{ datetime.fmt }}</time>
+                </div>
```

Comment:
If the suggestion above is followed, then we should directly test the datetime object, instead of the `iso` property to test if there's a valid object.
```suggestion
                <div class="time" v-if="start_datetime && end_datetime">
                    <time :datetime="start_datetime.iso">{{ start_datetime.fmt }}</time> to
                    <time :datetime="end_datetime.iso">{{ end_datetime.fmt }}</time>
                </div>
                <div class="time" v-else-if="datetime">
                    <time :datetime="datetime.iso">{{ datetime.fmt }}</time>
                </div>
```